web/packages/teleterm/src/ui/ModalsHost/modals/UsageData/UsageData.tsx

```diff
@@ -68,6 +68,16 @@ export function UsageData(props: UsageDataProps) {
             Do you agree to Teleport Connect collecting anonymized usage data?
             This will help us improve the product.
           </Text>
+          <Text typography="body1" color="text.secondary">
+            To learn more, see our{' '}
+            <Link
+              href="https://goteleport.com/docs/faq/#teleport-connect"
+              target="_blank"
+            >
+              documentation
+            </Link>
```

Comment:
I'd include "our" in the link as well. We're not really consistent here, I just grepped web for "documentation" (excluding test files) and we usually have "view our documentation". HelmChart and Connect's OnlineDocumentGateway link "the documentation" but there's also a stray place which links just "documentation" (Databases ConnectDialog).

"View our documentation" doesn't make sense in every context though (I think), so here "our documentation" would be fine.